pkg/master/master.go

```diff
@@ -206,18 +204,19 @@ func (c *Config) createLeaseReconciler() reconcilers.EndpointReconciler {
 	if err != nil {
 		glog.Fatalf("Error creating storage factory: %v", err)
 	}
-	endpointConfig, err := c.ExtraConfig.StorageFactory.NewConfig(kapi.Resource(DefaultStorageEndpoint))
+	endpointPrefix := c.ExtraConfig.StorageFactory.ResourcePrefix(api.Resource("endpoints"))
+	endpointConfig, err := c.ExtraConfig.StorageFactory.NewConfig(kapi.Resource(endpointPrefix))
 	if err != nil {
 		glog.Fatalf("Error getting storage config: %v", err)
 	}
 	endpointsStorage := endpointsstorage.NewREST(generic.RESTOptions{
 		StorageConfig:           endpointConfig,
 		Decorator:               generic.UndecoratedStorage,
 		DeleteCollectionWorkers: 0,
-		ResourcePrefix:          c.ExtraConfig.StorageFactory.ResourcePrefix(kapi.Resource(DefaultStorageEndpoint)),
+		ResourcePrefix:          c.ExtraConfig.StorageFactory.ResourcePrefix(kapi.Resource(endpointPrefix)),
```

Comment:
should be `ResourcePrefix: endpointPrefix`, right?

Reply:
Yes

Reply:
same response as above :)